lib/procore/errors.rb

```diff
@@ -28,6 +28,9 @@ def initialize(message, response: nil)
   # required parameters or sending a request to access a non-existent resource.
   InvalidRequestError = Class.new(Error)
 
+  # Raised when the request 403's
+  ForbiddenError = Class.new(Error)
```

Comment:
Hmm, do we possibly want just use `AuthorizationError` instead of introducing another class? I would consider `401` and `403` to be types of `AuthorizationError`s.

Reply:
I actually would prefer two separate error classes as it let's us handle the two cases separately.